File: Funciones/20. contador_for/js/contadoFor.js
/**
 * Contar hasta 5 con for
 * Autor: Santiago GM
 * Fecha: 03/04/2024
 */

let numero;

//Como paramétros 

function contar(pnumero){
    numero = pnumero;
    let contador;
    let limite = 5;
    let resultado = "";

    for(contador = 1; numero >= contador; contador ++){

        resultado += contador+".\n";
    }
    return resultado;
}

// Como Expresión

const contarExp = function (pnumero){
    numero = pnumero;
    let contador;
    let limite = 5;
    let resultado = "";

    for(contador = 1; numero >= contador; contador ++){

        resultado += contador+".\n";
    }
    return resultado;
}

File: python/20. contador_for/utils/contadoFor.py
# Contar hasta 5 con for
#Autor: Santiago GM

#Como paramétros

def contar_hasta_5(number):
    limite, resultado, contador = number, "", 1

    for i in range(contador, limite + 1):
        resultado += f'{i}\n'
    return resultado

#funcón lambda

contar_hasta_5_lamb =lambda number: '\n'.join([str(i) for i in range(1, number + 1)])